docs/resources/region.md

```diff
@@ -1,35 +1,55 @@
----
-# generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "biganimal_region Resource - terraform-provider-biganimal"
-subcategory: ""
-description: |-
-  Manage a region
----
-
 # biganimal_region (Resource)
-
-Manage a region
-
-
+The region resource is used to manage regions for a given cloud provider. See https://www.enterprisedb.com/docs/biganimal/latest/getting_started/activating_regions/ for more details.
+
+!> Creation of region resources is not supported for now.
+
+## Example Usage
+```terraform
+terraform {
+  required_providers {
+    biganimal = {
+      source  = "biganimal"
+      version = "0.3.1"
+    }
+  }
+}
+
+resource "biganimal_region" "this" {
+  cloud_provider = "aws"
+  region_id      = "eu-west-1"
+}
+
+output "region_status" {
+  value = resource.biganimal_region.this.status
+}
+
+output "region_name" {
+  value = resource.biganimal_region.this.name
+}
+
+output "region_continent" {
+  value = resource.biganimal_region.this.continent
+}
+```
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `cloud_provider` (String) Cloud Provider
-- `region_id` (String) Region ID
+- `cloud_provider` (String) Cloud Provider. e.g. "aws", "azure".
+- `region_id` (String) Region ID of the region. e.g. "germanywestcentral" in the azure cloud provider, "eu-west-1" in the aws cloud provider.
```

Comment:
```suggestion
- `region_id` (String) Region ID of the region. For example, "germanywestcentral" in the Azure cloud provider or "eu-west-1" in the AWS cloud provider.
```

Reply:
Spelling correction of the cloud providers, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/fd43c8ae86d3aad7abff08c0cece786c2e54a182
Adding `or`, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/460be1df55a563f5ac981ad7e7a2d33021b8e941